kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/clients/records/KafConsumerRecord.java

```diff
@@ -73,7 +73,7 @@ public static KafConsumerRecord parseFromJsonString(String response) {
             return objectMapper.readValue(response, KafConsumerRecord.class);
         }
         catch (JsonProcessingException e) {
-            LOGGER.atError().setMessage("Error: {}").addArgument(e.getMessage()).log();
+            LOGGER.atError().setMessage("Error: {}").addArgument(e).log();
```

Comment:
With the fluent api, you use `setCause`.

```suggestion
            LOGGER.atError().setMessage("Something bad happened").setCause(e).log();
```